app.js

```diff
@@ -0,0 +1,35 @@
+const body = document.querySelector("body");
+const firstPage = document.querySelector(".first-page");
+const introText = document.querySelector(".intro-text");
+const eventTitles = document.querySelector(".event-titles");
+// const eventKeyResult = document.querySelector(".key-result");
+// const eventCodeResult = document.querySelector(".code-result");
+// const eventWhichResult = document.querySelector(".which-result");
+
+document.addEventListener("keydown", (event) => {
+    console.log(event);
+    firstPage.style.display = "none";
+    
+    let eventKeyResult = event.key;
+    console.log(eventKeyResult);
+
+    let eventCodeResult = event.code;
+    console.log(eventCodeResult);
+
+    let eventWhichResult = event.which;
+    console.log(eventWhichResult)
+
+    let paraKey = document.querySelector(".keyResult");
+    paraKey.textContent = eventKeyResult;
+
```

Comment:
Do you prefer storing the event properties in variables rather than just passing them in directly to textContent?

Reply:
so I could do:

.keyResult.textContent = eventKeyResult;??

Reply:
I guess I meant more along the lines of `paraWhich.textContent = event.which` or even `document.querySelector(".whichResult").textContent = event.which`